css/selectors/view-transition-image-pair.json

```diff
@@ -4,8 +4,8 @@
       "view-transition-image-pair": {
         "__compat": {
           "description": "<code>::view-transition-image-pair()</code>",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::view-transition-image-pair",
-          "spec_url": "https://drafts.csswg.org/css-view-transitions/#selectordef-view-transition-image-pair-pt-name-selector",
+          "mdn_url": "https://drafts.csswg.org/css-view-transitions/#::view-transition-image-pair",
```

Comment:
Looks like you accidentally replaced the mdn_url here.

Reply:
oops, thanks for catching that — I just now pushed a fix